src/app/modules/design-patterns/models/roulette.model.ts

```diff
@@ -1,4 +1,4 @@
-export interface RouletteModel {
```

Comment:
change to `src/app/modules/design-patterns/observer/models/roulette-bet.model.ts`